cvat-ui/src/components/create-task-page/advanced-configuration-form.tsx

```diff
@@ -92,7 +92,7 @@ function validateURL(_: RuleObject, value: string): Promise<void> {
     return Promise.resolve();
 }
 
-const isInteger = ({ min, max }: { min?: number; max?: number }) => (
+const isInteger = ({ min, max, toBeSkipped }: { min?: number; max?: number; toBeSkipped?: number }) => (
```

Comment:
Consider renaming the `isInteger` function or adding another validator that checks for specific values.

Reply:
With this, the functionality is added with minimal modifications, and the function gets reused in consensus-configuration. The existing function also checks for the range and whether it's an integer, as the function's name suggests...